### PR TITLE
fix #7992 feat/test(nimbus): Add sentry to monitor our integration tests

### PR DESCRIPTION
**Pull request by jrbenny35**
Because:
* Tracking down errors within flakey integration tests can be quite difficult.

This commit:
* Adds a library to report test errors to sentry.

With sentrys rich logging and stacktraces, this will help us identify what is causing a flakey test much quicker than before.

### CODE REVIEW

**Comment by jrbenny35** [comment]
We just need to add ```PYTEST_SENTRY_DSN``` with the correct DSN to the circleci env variables.

**Comment by jrbenny35** [comment]
Screenshot
image

File: Makefile [hunk 1]
```diff
@@ -189,7 +189,7 @@ integration_test_legacy: build_prod
 	MOZ_HEADLESS=1 $(COMPOSE_INTEGRATION) run firefox sh -c "sudo apt-get -qqy update && sudo apt-get -qqy install tox;sudo chmod a+rwx /code/app/tests/integration/.tox;tox -c app/tests/integration -e integration-test-legacy $(TOX_ARGS) -- -n 2 $(PYTEST_ARGS)"
 
 integration_test_nimbus: build_prod
-	MOZ_HEADLESS=1 $(COMPOSE_INTEGRATION) run firefox sh -c "if [ "$$UPDATE_FIREFOX_VERSION" = "true" ]; then sudo ./app/tests/integration/nimbus/utils/nightly-install.sh; fi; firefox -V; sudo apt-get -qqy update && sudo apt-get -qqy install tox;sudo chmod a+rwx /code/app/tests/integration/.tox;tox -c app/tests/integration -e integration-test-nimbus $(TOX_ARGS) -- $(PYTEST_ARGS)"
+	MOZ_HEADLESS=1 $(COMPOSE_INTEGRATION) run firefox sh -c "if [ "$$UPDATE_FIREFOX_VERSION" = "true" ]; then sudo ./app/tests/integration/nimbus/utils/nightly-install.sh; fi; firefox -V; sudo apt-get -qqy update && sudo apt-get -qqy install tox;sudo chmod a+rwx /code/app/tests/integration/.tox;PYTEST_SENTRY_DSN=$(PYTEST_SENTRY_DSN) PYTEST_SENTRY_ALWAYS_REPORT=$(PYTEST_SENTRY_ALWAYS_REPORT) tox -c app/tests/integration -e integration-test-nimbus $(TOX_ARGS) -- $(PYTEST_ARGS)"
```
Comment (jaredlockhart) [inline Makefile:192]: Do we have to set `PYTEST_SENTRY_ALWAYS_REPORT` in Circle too?

Comment (jrbenny35) [reply Makefile:192]: No...so I left this in the makefile so we can easily debug. This just reports test failures to sentry, vs using the logic to only report tests than were rerun and then passed

**Review by jaredlockhart — APPROVED** [review]
Wow awesome!  Approved, and I already put a `SENTRY_DSN` into Circle env so it should be ready to go right away 🎉 Amazing thank you @jrbenny35 ! 🎉 🎉 🎉

**Pull request merged**